### PR TITLE
ui: Invoke the createTable function regardless of response

### PR DESCRIPTION
**Pull request by Ladsgroup**
Bug: T149118

### CODE REVIEW

File: ores/wsgi/static/js/scorer.js [hunk 1]
```diff
@@ -79,14 +79,14 @@ function getResults() {
 	});
 	models_url = models_url.slice(0, -1);
 	var url = "/scores/" + $('#wikiDropDownInput').attr('value') + "/?models=" + models_url + "&revids=" + revs;
-	$.get(url, function (data) {
+	$.get(url, datatype='jsonp').always(function(data) {
```
Comment (ricordisamoa) [inline ores/wsgi/static/js/scorer.js]: Why not e.g. `/* datatype= */ 'jsonp'` to avoid creating unused variable? :wink:

Comment (Ladsgroup) [inline ores/wsgi/static/js/scorer.js]: Done :)

**Comment by halfak** [comment]
There's a rendering issue where it seems the error pane isn't fully removed after an error occurs and a new revid is scored.  Not sure if it is related to this specific change or not.  

Steps reproduce:

1. Start up the dev server
2. Go to http://localhost:8080/ui/
3. Score the revid "-1" (and see error)
4. Score the revid "1" (success, but rows of table are far away from headers)

**Comment by Ladsgroup** [comment]
@halfak fixed. It's way cleaner now

File: ores/wsgi/static/js/scorer.js [hunk 2]
```diff
@@ -80,11 +80,11 @@ function getResults() {
...
 	var url = "/scores/" + $('#wikiDropDownInput').attr('value') + "/?models=" + models_url + "&revids=" + revs;
 	$.get({ url: url, datatype: 'jsonp' }).always(function(data) {
-		if ($('.table').length) {
-			$('.table').html(createTable(data));
-		} else {
-			$('#afterThis').after(createTable(data));
+		if ($('.alert').length) {
+			$('.alert').remove();
 		}
+		$('#tableContainer').remove();
```
Comment (halfak) [inline ores/wsgi/static/js/scorer.js]: I think maybe you want to clear the children -- not remove the container.

Comment (Ladsgroup) [inline ores/wsgi/static/js/scorer.js]: hmm, Let me see what I can do about it.

**Pull request merged**